aman/index.html

```diff
@@ -1,12 +1,12 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="overflow-hidden h-full">
```

Comment:
why this?

Reply:
Sir, this is used to manage the scrollbar behavior by restricting it to the body element.